### ISSUE TITLE
TopAppBarElevation don't work

### ISSUE THREAD

**Issue by raul19** [onebone/compose-collapsing-toolbar#8]
Hello!
I have a screen that the appbar changes its elevation through the scroll state, but it don't work on my tries, is possible to do it?

The problem I have found is that for the collapsing toolbar to work, the height of the composable is taken as expanded height, and that height is fixed in the examples because there is always an image or a background (Box) of fixed height, but in my case, the height has to be variable because the TopAppBar has to modify its height with the scroll and increase or decrease the elevation.

**Comment by onebone** [comment]
I didn't try it myself yet, but I will write some of my theoretical solution I've come up.

First thing to note is that the height of the collapsing toolbar is not fixed, and is variable depending on the user's scroll interaction. Only maximum and minimum heights are chosen by the children composables.

The reason I suspect why it does not work with TopAppBar  is that TopAppBar does not propagate scroll to the toolbar. 

If TopAppBar consumes nested scroll and flows it to the toolbar, then it will change its height accordingly.

**Comment by raul19** [comment]
Yes, sorry, I don't have explained it correctly, the maximum height is fixes by the composable height.
I use the CollapsingToolbarState.progress to change the AppBar height and the AppBar elevation, but the problem is that in this case, the height of the composable depends on the toolbar height, so the progress listener always returns 1 because the maximum height and the composable height is always the same.

**Comment by onebone** [comment]
Ah sorry, I have misunderstood your issue. To deal with a variable height composable, you can add a dummy composable with fixed height to set maximum and minimum height for now. 

I will add some sort of expandedHeight, collapsdHeight argument for CollapseToolbar in the next release.
